docs/UserGuide.md

```diff
@@ -135,19 +135,39 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+
+### Locating persons by tag: `filter`
+
+Finds persons whose names contain any of the given keywords.
+
+Format: `filter t/TAG [t/MORE_TAG]...`
+
```

Comment:
Might adding a tip to suggest having the filter to take in at least one tag help improve the user guide

Reply:
Ok, added it into the description of `filter`.